infra/storage/file_transfer_managers/s3Mgr.class.php

```diff
@@ -527,7 +527,7 @@ public function registerStreamWrapper()
 		$this->s3->registerStreamWrapper();
 	}
 	
-	public function getRemoteUrl($remote_file)
+	public function getRemoteUrl($remote_file, $expiry = null)
```

Comment:
the function overrides getRemoteUrl($remote_file) on kFileTransferMgr, i think we better align there the signature as well